app/javascript/pages/browse/MapBrowser.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <section class="MapBrowser">
+    <mapbox
+        access-token="pk.eyJ1IjoibXV0dWFsLWFpZC1hcHAiLCJhIjoiY2tmZTBvd3UwMDBhbTJ4cDlic2JmMWZoaiJ9.rWscBjdl1SMT5N0yekIJYg"
+        :map-options="{
+          style: 'mapbox://styles/mapbox/streets-v11',
```

Comment:
I don't have a strong preference for one style over another. For reference, i found this list:
https://docs.mapbox.com/api/maps/#mapbox-styles
Of those, i'd be good with `streets`, `light`, `satellite-streets`.
Curious if others have preferences. Also easy to change, so happy to go with your suggestion.